packages/graphql/src/schemaTypes/objectTypes/gql-RelevantRun.ts

```diff
@@ -33,5 +33,9 @@ export const RelevantRunSpecs = objectType({
     t.nonNull.int('completedSpecs', {
       description: 'Number of specs in the run that have finished being processed',
     })
+
+    t.dateTime('scheduledToCompleteAt', {
```

Comment:
Just curious re: use of `t.dateTime` - other object schemas seem to use `string` with an ISO-8601 timestamp value. Does `dateTime` auto-parse into a `Date`?

Reply:
I was matching the type coming from the Cloud schema.  It still ends up as a string in the generated types.